libafl/src/mutators/mutations.rs

```diff
@@ -989,13 +986,6 @@ where
             }
             Ok(MutationResult::Mutated)
         } else if first.end != size {
-            // The first range comes before the second range
-            debug_assert!(
```

Comment:
Are we sure? Better to keep the debug assert imho

Reply:
you have `NonZero::new(size - first.end).unwrap_unchecked()` a few lines later anyway